### PR TITLE
Feature/31999 parse migrations csv

### PR DESCRIPTION
**Pull request by danwilliams**

### CODE REVIEW

File: admin/README.md [hunk 1]
```diff
@@ -113,6 +113,16 @@ We use postgrator to run database migrations.
 
 `yarn migrate-sql` to migrate to the latest or `yarn migrate-sql 123` to migrate to a particular version
 
+### Parse school changes from CSV
```
Comment (jon-shipley) [inline admin/README.md]: What does this actually do?

File: admin/bin/parse-migrations-csv.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env node
+'use strict'
+
+require('dotenv').config()
+const csv = require('fast-csv')
+const fs = require('fs')
+const poolService = require('../services/data-access/sql.pool.service')
```
Comment (jon-shipley) [inline admin/bin/parse-migrations-csv.js]: This isn't needed anymore.

File: admin/bin/parse-migrations-csv.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+      value: name,
+      type: TYPES.NVarChar
+    })
+    params.push({
+      name: `oldDFENumber${i}`,
+      value: oldDFENumber,
+      type: TYPES.Int
+    })
+  })
+  return sqlService.modifyWithTransaction(queries.join('\n'), params)
```
Comment (jon-shipley) [inline admin/bin/parse-migrations-csv.js]: 👍

File: admin/bin/parse-migrations-csv.js [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+    console.log(`Updated ${schoolChanges} schools`)
+    process.exit(0)
+  } catch (error) {
+    throw error
+  }
+}
+
+main()
+  .then(() => {
+    poolService.drain()
```
Comment (jon-shipley) [inline admin/bin/parse-migrations-csv.js]: This method was moved to `sqlService.drainPool`

File: admin/bin/parse-migrations-csv.js (hunk 2 above)
Comment (GuyHarwood) [reply admin/bin/parse-migrations-csv.js]: @danwilliams i would suggest a refresh from master, because that file no longer exists (sql.pool.service)

**Comment by GuyHarwood** [comment]
@danwilliams tested and confirmed as working

**Pull request merged**